h5bp/ssl/policy_future.conf

```diff
@@ -0,0 +1,43 @@
+# ----------------------------------------------------------------------
+# | SSL policy - Future                                                |
+# ----------------------------------------------------------------------
+
+# For services that want to be on the bleeding edge, the parameters
+# below sacrifice compatibility for the highest level of security & performance
+#
+# (!) This policy enfore a strong SSL configuration, which may raise
+#     errors with old clients.
+#     If a more compatible profile is required, use intermediate policy.
+#
+# (1) The NIST curves (prime256v1, secp384r1, secp521r1) are known to be weak
+#      and potentially vulnerable.
+#
+#     Add them back to the parameter `ssl_ecdh_curve` below to support
+#     Microsoft Edge and Safari.
+#
+#     https://safecurves.cr.yp.to/
+#
+# (2) Supported by NGINX >=1.15.3. Enables TLS 1.3 0-RTT, allows for faster
+#     resumption of TLS sessions. Requests sent within early data are subject
+#     to replay attacks.
+#
+#     To protect against such attacks at the application layer, the
+#     $ssl_early_data variable should be used:
+#       proxy_set_header Early-Data $ssl_early_data;
+#
+#     The application should return response code 425 for anything that could
+#     contain user supplied data.
+#
+#     https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/425
+#
+# https://wiki.mozilla.org/Security/Server_Side_TLS#Recommended_configurations
```

Comment:
This Mozilla link is no more relevant, you can remove it.
You may add other related refs:

```suggestion
# https://github.com/mozilla/server-side-tls/issues/217
# https://github.com/certbot/certbot/issues/6367
```